english/selfie/selfie.md

```diff
@@ -1,3 +1,13 @@
 You take out your phone and take a selfie. You look at the picture.
 
-Oh no! You are made of marshmallow!
\ No newline at end of file
+Oh no! You are made of marshmallow!
+
+This sudden revelation has now complicated the matter.
+
+You hate marshmallows but can you really hate yourself?  
+
+The next logical step then is to: 
+
+[Begin to eat!](eating/inside.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.